tests/acceptance/features/bootstrap/Provisioning.php

```diff
@@ -1609,6 +1844,18 @@ public function userShouldBelongToGroup($user, $group) {
 		);
 	}
 
+	/**
+	 * @param string $group
+	 *
+	 * @return array
+	 */
+	public function getUsersOfALdapGroup($group) {
```

Comment:
```suggestion
	public function getUsersOfLdapGroup($group) {
```
for these methods that are not also gherkin steps, we do not need to bother put "a", "an", "the" in the method name.
That saves having t think if it should be `ALdapGroup` or `AnLdapGroup`

Reply:
done!